Reinterop~/Fields.cs

```diff
@@ -156,12 +163,38 @@ private static void GenerateSingleFieldAccessors(CppGenerationContext context, T
                 TypeDeclarationsReferenced: new[] { setType }
             ));
 
+            string[] invocation = new[]
+            {
+                $"auto result = Field_get_{field.Name}({string.Join(", ", interopGetParametersCall)});",
+                $"return {getType.GetConversionFromInteropType(context, "result")};"
+            };
+            if (hasStructRewrite)
+            {
+                if (fieldType.Kind == InteropTypeKind.Nullable)
```

Comment:
When I do a nullable non-struct field in a class (for example, I added `public int? test2` to `CesiumGeoreference`, it's not generating code with `std::optional`, it's doing this instead:
```
  public: ::DotNet::System::Nullable1<::std::int32_t> test2() const;
  public: void test2(const ::DotNet::System::Nullable1<::std::int32_t>& value) const;
```

Is that intentional?

Reply:
Not intentional, but I think it can be addressed in a separate PR. I wrote an issue for it:
https://github.com/CesiumGS/cesium-unity/issues/225